refinery/tool_manager/views.py

```diff
@@ -166,4 +164,4 @@ def relaunch(self, request, *args, **kwargs):
             return tool.launch()
         except Exception as e:
             logger.error(e)
-            return HttpResponseBadRequest(e)
+            return Response(e, status.HTTP_400_BAD_REQUEST)
```

Comment:
Why not keep using HttpResponseBadRequest?

Reply:
Thanks Ilya I missed this.
I'm still in the process of looking these changes over myself. No need to double the effort unless you're really motivated to look this over.

Reply:
I just assume that a PR is a final version (as opposed to just a new branch).

Reply:
I'm fairly sure we've decided before as a team that a PR is not ready until one assigns a reviewer. ( I assigned Jennifer on this now before I forgot because that is who Github requested, and because I knew Jennifer has already left for today) Meanwhile I'm going over it myself.

If you have the availability again to review team PRs on code you aren't a suggested reviewer on please let us all know.

Reply:
Right. I can't review all PRs but I do look over some if I happen to have downtime or need a context switch.